GSCLSP.Extension/README.md

```
@@ -1,17 +1,19 @@
 
-[Visual Studio Code](https://code.visualstudio.com/) language support for *Call of Duty®*'s scripting language `.gsc`, `.gsh`, `.csc`, and `.csh` files, powered by the **GSCLSP [C# language server](https://github.com/Lierrmm/GSCLSP/tree/main/GSCLSP.Server)**.
+[Visual Studio Code](https://code.visualstudio.com/) language support for _Call of Duty®_'s scripting language `.gsc`, `.gsh`, `.csc`, and `.csh` files, powered by the **GSCLSP [C# language server](https://github.com/Lierrmm/GSCLSP/tree/main/GSCLSP.Server)**.
 
 ## Features
 
 ### Syntax Highlighting
+
 - Adds colored keywording
 - "Dead code" zones for early returns & unused preprocessors
 <div align="center"> 
 <img src="https://raw.githubusercontent.com/Lierrmm/GSCLSP/refs/heads/main/GSCLSP.Extension/images/syntax1.png" width="60%">
```

markdownlint-cli2 (0.22.1)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)